### ISSUE TITLE
UIP-12 - Reduce Target Time for Blocks

### ISSUE THREAD

**Issue by iquidus** [ubiq/UIPs#12]
### Simple Summary

Reduce target time for blocks from 88seconds to 22seconds.

### Abstract

It's become evident that with block time and proof of work, there will always be a heavy variance and more greatly so when block time is increased. This is basically known as Poisson distribution and we have all encountered it with Ubiq (and Bitcoin) in the form of long blocks. Long blocks wouldn't be an issue if they were at a consistent time. Only really non-PoW consensus engines like PoS/PoA can give you that consistent timing. With a shorter block time, users do not perceive any delay but also proof of work decreases its realistic time value of finding blocks.

This UIP proposes changing the target time to 22 seconds (4x faster) than now. In this scenario a "long block" would probably take 1 minute. A rare, extremely long block 2 mins.

We would also remove Flux and rely on ETH's difficulty algorithm which will make initial sync after the Flux blocks extremely fast (literally minutes) due to significantly reduced disk IO.

Our initial parameters have been great for bootstrapping the network, resulting in minimal bloat while infrastructure and use cases have been built out, but now with the Ubiq DeFi ecosystem growing these initial parameters have served their purpose. It is now time to prioritize speed and UX over bloat reduction. Latest generation Raspberry Pi nodes will have no issues handing this increase in block time speed.

### Specification

TODO

### Implementation

Adoption of the content of this UIP requires a hard fork as it introduces changes that are not backward compatible. 

Due to the nature of this proposal it will go through Escher governance.

### Proposed Timeline

TBD

**Comment by fafrd** [comment]
As someone involved with the https://token.gallery NFT platform, I am in favor of this change. Long waits for transactions are one of the primary complaints from newcomers who aren't used to the Ubiq blockchain. The current wait time, plus the requirement for 2 confirmations (as suggested by Ubiq developers) results in typical wait times of about 3 minutes for our platform.

This change would be a large UX improvement for any dapps running on Ubiq, and is in line with user's expectations from other platforms like Ethereum and Polygon.

**Comment by Bullchad123** [comment]
I think this proposal is very much needed and has broad support of the Ubiq community as per numerous discord conversations

To be a serious contender in the DeFi and NFT space, shorter blocktime are increasingly a necessity. The future of the platform is about attracting new users and new developers, so I support this UIP

**Comment by ZenithII** [comment]
In my opinion the change to a lower blocktime is a welcome development. Most people in the space expect fast transactions, with the 88 second blocktime it is a little longer wait on average. But random blocks taking a few minutes does not meet the expectation of many DeFi users. Imho this needs to be implemented ASAP for Ubiq to close the gap on alternative chains and also meet the demands/expectations users have coming from other chains!

**Comment by LessDaStress** [comment]
When I first got interested in Ubiq back in 2018, one of the features that caught my attention was the Flux Algorithm. 
(https://blog.ubiqsmart.com/ubiq-research-the-flux-difficulty-algortihm-561507950339) It was one of the features that sets Ubiq apart from the other Dagger Hashimoto based blockchain. Back then, my only use case for Ubiq was mining it. With my limited knowledge of blockchain, my understanding was that 88 secs block time is a good trade off for network stability.  Purpose of Ubiq was to offer an enterprise-stable platform.

But the blockchain scene has changed a lot since then. Competing blockchain, Layer-2, NFTs, public use DefI etc. And now that I have actually interacted with some of the tools and projects available on Ubiq (Shinobi Ninja, TokenGallery, wallets etc), I think there's a good reason to consider re-evaluating the block time.  

88 secs is a bit too long under current scenario. As this UIP states, now is time prioritize speed and UX over bloat reduction. 22 secs is good trade off between speed and stability.  So I'm all in for this UIP.

**Comment by skywalk3rUBQ** [comment]
YES
Fits need while not being too fast, great thought out compromise.

**Comment by BrawneUBQ** [comment]
I wasn't originally a fan of the decreased block times in tandem with the increased rewards, as I had inflation concerns. However, after reviewing various feedback/comments in Discord, I have come around. I'm fine with the current proposals.

**Comment by LessDaStress** [comment]
> I wasn't originally a fan of the decreased block times in tandem with the increased rewards, as I had inflation concerns. However, after reviewing various feedback/comments in Discord, I have come around. I'm fine with the current proposals.

If EIP-1559 is not part of the UIPs, I probably will have to think a lot more about the impact of reducing the block time. I believe EIP-1559 with help with the inflation rate.

**Comment by j40j** [comment]
I support this change in order to be more competitive with other chains. The current block time is way too long and considered a deterrent when interacting with NFTs or transacting on Shinobi. I support 22 seconds in that there is still enough time for finding blocks.

**Comment by hekkort** [comment]
Admittedly, I haven't done too much in terms of defi usage on UBQ. But buying for example on token.gallery is from a UX pov a poor exp, which has also already been mentioned above, I am in favor of this change to improve UX.
